platform/admin/core/admin-core-appservice/src/test/java/org/codice/ddf/admin/application/service/impl/ApplicationServiceBeanTest.java

```diff
@@ -236,6 +244,48 @@ public void testInstallProfile() throws Exception {
     assertThat(options, hasItem(FeaturesService.Option.NoAutoRefreshBundles));
   }
 
+  @Test
+  public void testInstallFeatureCallIsPrivileged() throws Exception {
+    final AtomicReference<StackTraceElement[]> stackTraceElements = new AtomicReference<>();
+
+    doAnswer(
+            AdditionalAnswers.answerVoid(
+                o -> stackTraceElements.set(Thread.currentThread().getStackTrace())))
+        .when(mockFeaturesService)
+        .installFeature(anyString(), any(EnumSet.class));
+
+    ApplicationServiceBean serviceBean =
+        new ApplicationServiceBean(
+            testAppService, testConfigAdminExt, mBeanServer, mockFeaturesService);
+    serviceBean.installFeature("profile-name");
+
+    assertThat(
+        Arrays.stream(stackTraceElements.get())
+            .anyMatch(e -> e.toString().startsWith(DO_PRIVILEGED_STACK_ELEMENT)),
```

Comment:
❗️ This relies on the `toString()` returning a given format. You would be better of testing the classname separately. As for testing the test method, I think it is sufficient to check if the AccessController is on the stack.